client/my-sites/plans/jetpack-plans/product-store/bundle-list/index.tsx

```diff
@@ -93,10 +71,58 @@ export const BundlesList: React.FC< BundlesListProps > = ( {
 		);
 	} );
 
+	const allItems = useMemo(
+		() => [ ...popularItems, ...otherItems ],
+		[ otherItems, popularItems ]
+	);
+
 	return (
 		<div className="jetpack-product-store__bundles-list">
 			<MostPopular heading={ translate( 'Most popular bundles' ) } items={ mostPopularItems } />
 			<SeeAllFeatures />
+
+			{ /* Show All items only if there is something in otherItems */ }
+			{ otherItems.length ? (
```

Comment:
Is it possible to make this chunk of codes reusable component so we can also use this in the product list?

Reply:
Yes, that's a good observation. I even started to do that in this PR but then decided to do it in a separate PR.